### PR TITLE
Allow FieldTable definition to use an existing database table

### PR DESCRIPTION
**Pull request by xybrnet**
@longhotsummer

### CODE REVIEW

File: wazimap/data/tables.py [hunk 1]
```diff
@@ -364,7 +376,7 @@ def build_models(self):
                 model.data_table = self
```
Comment (xybrnet) [inline wazimap/data/tables.py]: ... I don't know how to comment on collapsed code : /

Anyway, should we add the `existing_table` argument to the above too, when calling for each level?

Comment (xybrnet) [reply wazimap/data/tables.py]: Something like this:
```db_table = None
      if self.db_table:
          db_table = '%s_%s' % (self.db_table, level)
      model = self._build_model_from_fields(self.fields, self._table_name(level), level, value_type=self.value_type, existing_table=db_table)
```
I'm just not sure how to properly test this.

File: wazimap/data/tables.py [hunk 2]
```diff
@@ -547,13 +559,15 @@ def permute(level, field_keys, rows):
 
         return data
 
-    def _build_model_from_fields(self, fields, table_name, geo_level=None, value_type=Integer):
+    def _build_model_from_fields(self, fields, table_name, geo_level=None, value_type=Integer, existing_table=None):
```
Comment (longhotsummer) [inline wazimap/data/tables.py]: What if we just pass in ``existing_table`` as ``table_name`` when we call this function? That way it'll return the cached model (hopefully that works ok).

Comment (xybrnet) [reply wazimap/data/tables.py]: We could. I had it like that, but then we need to change some of the references from `table_name` to `self.id.lower()`.

Like line 575 + 576.

@longhotsummer

**Comment by longhotsummer** [comment]
This looks fine, except for the scattered use of `model.data_table`, which is now ambiguous. If people setup two tables that have different options for total_column or denominator_key, then this is going to act weirdly.

**Comment by longhotsummer** [comment]
I've made a change to remove the heavy reliance on `model.data_table`

**Comment by xybrnet** [comment]
Wow. Nice 👍

**Pull request merged**